courses/machine_learning/time_series_prediction/modeling_bqml.ipynb

```diff
@@ -0,0 +1,1162 @@
+{
```

Comment:
<p>This will give us a more precise prediction rather than just predicting if the stock will go up, down, or stay the same. Thus we can treat this problem as either a regression problem or a classification problem, depending on the business needs.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/624/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>good comment. Added that as well. </p>